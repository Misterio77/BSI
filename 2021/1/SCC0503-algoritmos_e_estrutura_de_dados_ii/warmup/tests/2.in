data/reg.dat
2
