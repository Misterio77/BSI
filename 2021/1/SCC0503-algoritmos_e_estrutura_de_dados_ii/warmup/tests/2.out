nUSP: 1
Nome: Winter Y. Hodges
Curso: Engenharia da Computacao
Nota: 5.00

nUSP: 2
Nome: Doris S. Mathis
Curso: Sistemas de Informacao
Nota: 5.00

nUSP: 3
Nome: Vernon U. Dunlap
Curso: Sistemas de Informacao
Nota: 0.00

nUSP: 4
Nome: Duncan D. Peters
Curso: Ciencia da Computacao
Nota: 1.00

nUSP: 5
Nome: Felix L. Landry
Curso: Engenharia da Computacao
Nota: 1.00

nUSP: 6
Nome: Keane P. Knapp
Curso: Sistemas de Informacao
Nota: 9.00

nUSP: 7
Nome: Linus Q. Mccoy
Curso: Estatistica
Nota: 4.00

nUSP: 8
Nome: Quyn T. Rowland
Curso: Sistemas de Informacao
Nota: 9.00

nUSP: 9
Nome: Geraldine V. Reilly
Curso: Ciencia da Computacao
Nota: 1.00

nUSP: 10
Nome: Knox V. Floyd
Curso: Ciencia da Computacao
Nota: 10.00

nUSP: 11
Nome: Deacon V. Olsen
Curso: Ciencia da Computacao
Nota: 2.00

nUSP: 12
Nome: Mara P. Schneider
Curso: Engenharia da Computacao
Nota: 3.00

nUSP: 13
Nome: Patience V. Bullock
Curso: Engenharia da Computacao
Nota: 8.00

nUSP: 14
Nome: Charles E. Snider
Curso: Engenharia da Computacao
Nota: 3.00

nUSP: 15
Nome: Mariko D. Gaines
Curso: Estatistica
Nota: 6.00

nUSP: 16
Nome: Nola D. Rocha
Curso: Engenharia da Computacao
Nota: 4.00

nUSP: 17
Nome: Brennan C. Zimmerman
Curso: Ciencia da Computacao
Nota: 4.00

nUSP: 18
Nome: Aurelia S. Hess
Curso: Sistemas de Informacao
Nota: 7.00

nUSP: 19
Nome: Cassidy E. Mack
Curso: Sistemas de Informacao
Nota: 0.00

nUSP: 20
Nome: Shoshana B. Carey
Curso: Estatistica
Nota: 2.00

nUSP: 21
Nome: Myra Y. Bailey
Curso: Sistemas de Informacao
Nota: 3.00

nUSP: 22
Nome: Garth J. Shepard
Curso: Estatistica
Nota: 6.00

nUSP: 23
Nome: Maile I. Henry
Curso: Engenharia da Computacao
Nota: 10.00

nUSP: 24
Nome: Sigourney H. Garrett
Curso: Estatistica
Nota: 9.00

nUSP: 25
Nome: Aaron W. Dean
Curso: Estatistica
Nota: 3.00

nUSP: 26
Nome: Brittany P. Cline
Curso: Engenharia da Computacao
Nota: 8.00

nUSP: 27
Nome: Amy L. Witt
Curso: Sistemas de Informacao
Nota: 6.00

nUSP: 28
Nome: Brenna R. Nguyen
Curso: Engenharia da Computacao
Nota: 2.00

nUSP: 29
Nome: Aladdin J. Henson
Curso: Sistemas de Informacao
Nota: 0.00

nUSP: 30
Nome: Allen M. Henry
Curso: Sistemas de Informacao
Nota: 5.00

nUSP: 31
Nome: Jade W. Richardson
Curso: Sistemas de Informacao
Nota: 4.00

nUSP: 32
Nome: Alfonso J. Decker
Curso: Ciencia da Computacao
Nota: 9.00

nUSP: 33
Nome: Jana O. Carver
Curso: Engenharia da Computacao
Nota: 2.00

nUSP: 34
Nome: Ora G. Joyce
Curso: Engenharia da Computacao
Nota: 2.00

nUSP: 35
Nome: Sade E. Horton
Curso: Engenharia da Computacao
Nota: 10.00

nUSP: 36
Nome: Quentin I. Wilder
Curso: Sistemas de Informacao
Nota: 5.00

nUSP: 37
Nome: Kennedy K. Winters
Curso: Ciencia da Computacao
Nota: 2.00

nUSP: 38
Nome: Jack G. Juarez
Curso: Estatistica
Nota: 1.00

nUSP: 39
Nome: Kimberley R. Phelps
Curso: Ciencia da Computacao
Nota: 0.00

nUSP: 40
Nome: Jennifer O. Dalton
Curso: Estatistica
Nota: 3.00

nUSP: 41
Nome: Xaviera N. Mclean
Curso: Ciencia da Computacao
Nota: 5.00

nUSP: 42
Nome: Violet V. Potter
Curso: Sistemas de Informacao
Nota: 7.00

nUSP: 43
Nome: Buckminster F. Gallegos
Curso: Engenharia da Computacao
Nota: 6.00

nUSP: 44
Nome: Martha S. Collier
Curso: Sistemas de Informacao
Nota: 8.00

nUSP: 45
Nome: Jordan X. Hill
Curso: Sistemas de Informacao
Nota: 7.00

nUSP: 46
Nome: Hadassah T. Carver
Curso: Engenharia da Computacao
Nota: 5.00

nUSP: 47
Nome: Ginger J. Nielsen
Curso: Ciencia da Computacao
Nota: 4.00

nUSP: 48
Nome: Jin K. Pena
Curso: Engenharia da Computacao
Nota: 9.00

nUSP: 49
Nome: Maite R. Klein
Curso: Sistemas de Informacao
Nota: 2.00

nUSP: 50
Nome: Rachel X. Dorsey
Curso: Sistemas de Informacao
Nota: 9.00
